config/env/default.js

```diff
@@ -28,5 +28,8 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUploadDir: './modules/users/client/img/profile/uploads/'
```

Comment:
@ilanbiala I think Multer requires it. If you look at the Express config diff, you'll see it was there. 

If that's the case, then we either have to add the *./* here, or add it here https://github.com/meanjs/mean/pull/948/files#diff-028054ee3eec2278e53157ccf4668fe4R59

@gustavodemari Maybe add a comment to the code to make it clear.

This way may be cleaner. WDYT?